### DISCUSSION TITLE
about precise position problem

### DISCUSSION THREAD

**Discussion by HuJunnj** [openTCS/opentcs#134] [General]
Can the coordinate accuracy be controlled in the micron level, it seems that only millimeter is supported at present, and does not support floating point type, we have relatively high accuracy requirements for the position, where can I set this function?

thanks in advance!

Best Regards
Jun

**Comment by swltr** [comment]
@HuJunnj Right now, you can only work with millimeters. openTCS was originally designed as a control system software for AGVs, where sub-millimeter precision is unexpected. Few AGV types have actually reached a precision of a centimeter in the past.

I'm curious: what is your use case, and what kind of vehicles do you use that actually achieve such high-precision positioning?
